driver/src/main/java/eu/cloudnetservice/cloudnet/driver/module/DefaultModuleProvider.java

```diff
@@ -58,10 +64,23 @@
   protected ModuleProviderHandler moduleProviderHandler;
   protected ModuleDependencyLoader moduleDependencyLoader;
 
+  /**
+   * Creates a new DefaultModuleProvider by calling {@link DefaultModuleProvider#DefaultModuleProvider(Path,
+   * ModuleDependencyLoader)} with the default library directory {@link DefaultModuleProvider#DEFAULT_LIB_DIR} and the
+   * default dependency loader {@link DefaultModuleProvider#DEFAULT_DEP_LOADER}.
```

Comment:
This is too noisy imho. Just describe what the method is doing and add a paragraph describing the equivalent call to the method.